smithy-model/src/test/java/software/amazon/smithy/model/loader/InvalidSmithyModelLoaderRunnerTest.java

```diff
@@ -49,11 +49,13 @@ public void testParserRunner(String file) {
                     .unwrap();
             throw new IllegalStateException("Expected a parse error for " + file);
         } catch (RuntimeException e) {
-            if (!e.getMessage().contains(expectedError)) {
+            String actualMessage = e.getMessage().replace("\n", "\\n");
```

Comment:
Eh -- that's not a public API and I don't want to open it up just yet. This is also not using the same test runner since it's parsing a comment to determine the expected parse error.